index.html

```diff
@@ -31,6 +31,192 @@ <h3>I am a Software Developer</h3>
 		 
 		</div>
 	</section>
-    
+  <section>
+		<div class="works">
+			<h2 class="recent">My Recent Works</h2>
+			<img src="Assets/backgroundcard.png" alt="">

```

Comment:
- [ ] The section must have the proper background image (decorations are always background images, not `img` tags), if an image is not part of the content (aesthetic reasons) use CSS background-image. [Check this link](https://stackoverflow.com/questions/492809/when-to-use-img-vs-css-background-image) for more info